Dockerfile

```diff
@@ -5,13 +5,15 @@ WORKDIR /usr/src/app
 
 COPY package*.json ./
 
-RUN npm ci --production
-
-USER node
+RUN npm install
 
 COPY . .
 
+RUN /usr/src/app/node_modules/typescript/bin/tsc -p tsconfig.json
```

Comment:
once you have added a buikld script to the package.json you should be able to change this to `npm run build`